composition/src/errors/errors.ts

```diff
@@ -1051,7 +1051,13 @@ export function invalidRootTypeFieldResponseTypesEventDrivenErrorMessage(
 
 export const invalidNatsStreamInputErrorMessage =
   `The "streamConfiguration" argument must be a valid input object with the following form:\n` +
-  `  input edfs__NatsStreamConfiguration {\n    consumerName: String!\n    streamName: String!\n  }`;
+  `  input edfs__NatsStreamConfiguration {\n    consumerInactiveThreshold: Int! = 30\n    consumerName: String!\n    streamName: String!\n  }`;
+
+export function invalidArgumentValueErrorMessage(hostCoords: string, kind: Kind, value?: string): string {
+  return (
+    `The value ` + (value ? `"${value}" ` : ``) + `passed to "${hostCoords}" is not type ${kindToTypeString(kind)}.`
```

Comment:
quotations around "${kindToTypeString}" is the standard :D

Reply:
Ops, now it should be ok, thanks!